.gitignore

```diff
@@ -59,3 +59,6 @@ rpc/client.gen.go
 
 # Ignored generated files by build/config/main.go
 build/config/gen*.json
+
+# Ignore keybase files
+**/.keybase/*
```

Comment:
Should this be part of the lowercase functions PR?

Reply:
It's not necessary. I just had the `keybase` folder in my local when I worked on the project. But I just just remove that folder and the gitignorecode for a more clean PR